packages/components/src/components/button/button.scss

```diff
@@ -84,20 +80,20 @@
             }
         }
         &--red {
-            background: var(--button-primary-default);
+            background: var(--status-danger);
         }
         &--green {
-            background: $COLOR_GREEN_1;
+            background: var(--status-success);
```

Comment:
@zaki-binary can you please update it for both primary and secondary styles

Reply:
As discussed, we'll do this in `Button` refactoring.